consensus/vbft/block_pool.go

```diff
@@ -755,3 +756,27 @@ func (pool *BlockPool) onBlockSealed(blockNum uint32) {
 		delete(pool.candidateBlocks, n)
 	}
 }
+
+func (pool *BlockPool) getExecMerkleRoot(blkNum uint32) (common.Uint256, error) {
+	pool.lock.RLock()
+	defer pool.lock.RUnlock()
+	return pool.chainStore.getExecMerkleRoot(blkNum)
+}
+
+func (pool *BlockPool) getExecWriteSet(blkNum uint32) *overlaydb.MemDB {
+	pool.lock.RLock()
+	defer pool.lock.RUnlock()
+	return pool.chainStore.getExecWriteSet(blkNum)
+}
+
+func (pool *BlockPool) submitBlock(blkNum uint32) error {
+	pool.lock.RLock()
+	defer pool.lock.RUnlock()
+	return pool.chainStore.submitBlock(blkNum)
+}
+
+func (pool *BlockPool) getBlock(blockNum uint32) (*Block, error) {
```

Comment:
block pool has getSealedBlock, what's their difference?

Reply:
no difference.I will del this function later.